### PR TITLE
C_CPP: Introduction to C

### LINKED ISSUE #850: C_CPP: Introduction to C
<hr>

## Description 📜

I want to contribute using videography on the following:

1. Introduction to C (History of C and its uses/advantages)
2. Programming components, Structure & Keyword 

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

### PR DESCRIPTION
**Pull request by prettylittleminds**
<hr>

## Description 📜
_As a newbie in the field of tech, the first-ever programming language I learnt was C; hence I wish to contribute to this domain._

**C** as a language has evolved immensely over time. It was developed at **Bell Labs** by **Dennis Ritchie** between _1972 and 1973._ 
Ever since it has only been evolving with the times. 

It is still used in today's world to a great extent, because of its many irreplaceable features! Thus with this contribution, I've given an overview of C and its programming components, structure, and keywords. I hope it gives you a kickstart in your coding journey. 
**Happy coding! :)**

Fixes #850 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by ipsitagoel** [comment]
@Suvraneel It's good to go. You can have a look.

**Review by gunjan-g — CHANGES_REQUESTED** [review]
@prettylittleminds add the index line properly in readme along with relative link of file and link of video

File: C_CPP/IntroToC.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+It is still used in today's world to a great extent, because of its many irreplaceable features! Some of which include:
+- Fast speed and high efficiency
+- Comprises of rich set of built-in functions
+- Highly portable
+- Highly extensible
+- Highly structured
+
+That's why, using C today is still relevant and does wonders for a newbie!
+## Here's a video giving an overview of C:
+Link: https://drive.google.com/file/d/1pkg6cWkqxc3lnq7GV1bYUaAU4o4YcYNv/view?usp=sharing
```
Comment (Suvraneel) [inline C_CPP/IntroToC.md:13]: Make relative link with embedded picture

File: C_CPP/readme.md [hunk 2]
```diff
@@ -1 +0,0 @@
-
```
Comment (Suvraneel) [inline C_CPP/readme.md:1]: Add relative link C_CPP/Fundamentals/Introduction to C/readme.md

**Review by Suvraneel — CHANGES_REQUESTED** [review]
Shify to directory:
C_CPP/Fundamentals/Introduction to C/

File: C_CPP/readme.md (hunk 2 above)
Comment (prettylittleminds) [reply C_CPP/readme.md:1]: Sir there is an alrdy existing link with the name Introduction To C (documentation) in your mentioned path, but I've made a video contribution, for the same name i.e. Introduction To C. So I wanted to ask do I need to add a relative link to the alrdy existing one or make a new one?

**Comment by prettylittleminds** [comment]
Updated PR with requested changes.

**Comment by Suvraneel** [comment]
@prettylittleminds  LGTM ❤️

**Pull request merged**